### ISSUE TITLE
Group 3: Early Women Graduates using Tableau Public

### ISSUE THREAD

**Issue by naomiwells85** [SASDigitalHumanitiesTraining/Visualisation#27]

**Comment by CathGalway** [comment]
I'm not getting very much out of this at all. One of the things that is slowing me down is the fact that in one instance 1905 seems to have been entered as 1095. Do we think it's OK to correct this?

**Comment by ElisaCas** [comment]
Me neither. I think the csv file contains data not sorted correctly. For example dates, names + school, NUL, unmatched dates. What can we do with these data and file? Can we modify them in order to achieve the most appropriate and relevant visualization? Thanks

**Comment by EBenbow1** [comment]
Dear all, 

Very much the same for me, I also found the 1095 instance, I tried to do a graph of entry by year but it ended up looking a bit weird because a couple of the odd datasets. 

https://public.tableau.com/profile/eliot.benbow#!/vizhome/RHULEarlyGraduates/Sheet5

Best wishes,

Eliot

**Comment by CathGalway** [comment]
I think we're going to have to modify this data if we want it to display meaningfully.

**Comment by ElisaCas** [comment]
So do you think that data like NULL and random conjunctions like "and" are not useful and can be eliminate? I think so, but we are also modifying the original dataset. Probably this exercise was made with this purpose, to verify what happens in case the cvs file is not "perfect". Thank you.
Regards,
Elisa

**Comment by CathGalway** [comment]
Yes, I think we can do that, although you are right that it modifies the dataset. I'm interested in what we do with the "staff" category, which is entered in many columns, standing for date, school, subject etc. Also, see the eigencentrality column? Presumably it's used because the dataset is conceived as a social network, but we don't see the connections on which those numbers are based. What is giving me some concern though, is that often those numbers are attached to graduates coded as "staff" so by removing those people would we be taking out people who are significant in other ways to the dataset?

**Comment by ElisaCas** [comment]
Yes, probably we will lose some data about people. But the category Student is repeated and somehow redundant: full name+degrees etc. and then again names, surnames. About the eigencentrality, I can't understand its meaning, it looks like a math variable. I hope we can come up with something interesting by tomorrow!

**Comment by CathGalway** [comment]
Agreed. I'm going to spend half an hour today cleaning up this data - adding location to the schools and so forth. I'll post the results here. It'll be in around 90 minutes, because I'm teaching before then....

**Comment by EBenbow1** [comment]
This has been very interesting to think about, I've also been looking at the varied ways the "Degree" column for subjects and (sometimes) classification are displayed. There must be about 5 or 6 different combinations of data organisation and input going on there. Some give more information than others - specific subject, classification etc. I'll try and put something together for that this afternoon :)

**Comment by CathGalway** [comment]
I think I'm going to take out all the staff. On the grounds that it is unclear that they did a degree. I think the original data has been all the women in the college at the time. I'll keep them on one sheet, so you can add them back if you want.

**Comment by MarkQUB** [comment]
I have done the same and excluded 'staff'. I have done one looking at degrees by year so thought it unnecessary to include staff: https://public.tableau.com/profile/mark.o.rawe#!/vizhome/DegreesbyYear2/Sheet1?publish=yes

**Comment by MarkQUB** [comment]
> I have done the same and excluded 'staff'. I have done one looking at degrees by year so thought it unnecessary to include staff: https://public.tableau.com/profile/mark.o.rawe#!/vizhome/DegreesbyYear2/Sheet1?publish=yes

Apologies that is the wrong link! Here is the correct one: https://public.tableau.com/profile/mark.o.rawe#!/vizhome/DegreesbyYear3/Sheet1?publish=yes

**Comment by CathGalway** [comment]
This is taking longer than half an hour. Apologies.

**Comment by CathGalway** [comment]
![image](https://user-images.githubusercontent.com/75667009/101630563-f04beb00-3a1a-11eb-9eea-110b701c4729.png)
Here's a screenshot of what I've been able to do with it. I'm hoping I can go back and play about with filters.

**Comment by ElisaCas** [comment]
Dear All,

I am attaching the screenshots I have made in Tableau. I hope they work. I used the CVS file provided for the course, just slightly altered. The first one still contains data not really relevant, the others can be more interesting, because I didn't take into account data as NULL, "and". Some of them just show the same data by different visualizations: I think bar charts/bubbles/etc. and different colours can influence our perception of data and be more or less effective for our purposes. I also experimented, like in Tableau 5, where I considered Eingencentrality (but I don't know its importance or use) and name+subjects without changing the original file. 

Regards,
Elisa

![Tableau_first simulation](https://user-images.githubusercontent.com/75629359/101637889-a15b8100-3a2d-11eb-8ba9-49e90a7cc2d6.png)
![Tableau1](https://user-images.githubusercontent.com/75629359/101637893-a1f41780-3a2d-11eb-9d27-07b1f1e29435.png)
![Tableau2](https://user-images.githubusercontent.com/75629359/101637894-a28cae00-3a2d-11eb-8377-30e022025464.png)
![Tableau3](https://user-images.githubusercontent.com/75629359/101637897-a28cae00-3a2d-11eb-961f-f37d4e609cd6.png)
![Tableau4](https://user-images.githubusercontent.com/75629359/101637899-a3254480-3a2d-11eb-9911-01d5fc7fb78e.png)
![Tableau5](https://user-images.githubusercontent.com/75629359/101637900-a3254480-3a2d-11eb-8f21-6d280b0b9374.png)
![Tableau6](https://user-images.githubusercontent.com/75629359/101637901-a3bddb00-3a2d-11eb-95e4-bf21fbfc5538.png)

**Comment by CathGalway** [comment]
I've managed to get these three quite nicely linked visualizations, but I can't get them to save so they're all visible at the same time, so I can only send this screenshot. I'm afraid tomorrow is a bit of a nightmare for me, so I think this is as far as I can take this before the workshop. 
![image](https://user-images.githubusercontent.com/75667009/101671974-2904b800-3a4d-11eb-841e-3a98a577de9e.png)

**Comment by CathGalway** [comment]
So this is as far as I think I can take this before the workshop tomorrow, because of my schedule. I can't get it to save on the Tableau Public site so that it is viewable on one screen, but they are actually linked, which means you can examine the spikes and trends.

![image](https://user-images.githubusercontent.com/75667009/101672104-59e4ed00-3a4d-11eb-9cc8-0be19a140771.png)

**Comment by CathGalway** [comment]
I don't know whether this works https://public.tableau.com/profile/catherine.emerson#!/vizhome/TestonGraduates/Dashboard1?publish=yes

**Comment by ElisaCas** [comment]
Thank you Catherine. It works, it's interactive (from my smartphone, at least). I think is very clear.

**Comment by MarkQUB** [comment]
https://public.tableau.com/profile/mark.o.rawe#!/vizhome/GraduatesxArrivedxSubjectAreaxDegree/Sheet2?publish=yes

I was interested in looking at subject areas by the degrees held by graduates. This is all still very new to me so I apologise for the very rudimentary charts!
